### PR TITLE
Add deplo.io as sponsor

### PR DESCRIPTION
**Pull request by rnestler**

### CODE REVIEW

File: sps20/content/contents.lr [hunk 1]
```diff
@@ -24,6 +24,18 @@ The conference will be held in English and is suited for all skill levels, from
 
 The summit is held on the **OST campus** in the beautiful town of **Rapperswil**. Find more information on the [venue](/venue) page.
 
+### Sponsors
+
+#### Gold
+
+#### Silver
+
+<a href="https://deplo.io"><img src="/sponsors/deploio.svg" style="height: 80px; width: 80px"></a>
```
Comment (rnestler) [inline sps20/content/contents.lr:33]: Our CSS for images is a bit wild in `sps20/assets/scss/style.scss` so I have to set explicit `width`.

**Comment by turbotimon** [comment]
didn't we say, that the sponsors will be at the bottom?

File: sps20/content/contents.lr (hunk 1 above)
Comment (turbotimon) [reply sps20/content/contents.lr:33]: yes, thats true. i have a stall branch where i made some improves. i will try to fix things i  general when find time. but for this, i think its's fine!

**Comment by rnestler** [comment]
> didn't we say, that the sponsors will be at the bottom?

I don't remember it that way. For me it made sense to have it between the important information like location and the boilerplate stuff like "Non-Profit" and "Code of Conduct".

File: sps20/content/contents.lr [hunk 2]
```diff
@@ -24,6 +24,18 @@ The conference will be held in English and is suited for all skill levels, from
...
+
+#### Gold
+
+#### Silver
+
+<a href="https://deplo.io"><img src="/sponsors/deploio.svg" style="height: 80px; width: 80px"></a>
+
+#### Bronze
+
+Do you want to support Swiss Python Summit? Find out more about [sponsoring opportunities](./sponsoring#sponsoring-2024).
```
Comment (turbotimon) [inline sps20/content/contents.lr:37]: Maybe put this sentences after the ### Sponsors and not under #### Bronze ? But I if you have it there for a reason, let it as it is and merge it

**Comment by turbotimon** [comment]
> I don't remember it that way. For me it made sense to have it between the important information like location and the boilerplate stuff like "Non-Profit" and "Code of Conduct".

Ok, I saw once "at the bottom" but the last recent was "after location" in the meeting notes. So it's fine!

File: sps20/content/contents.lr (hunk 2 above)
Comment (rnestler) [reply sps20/content/contents.lr:37]: The reasoning was to have it in the end after all sponsors, but I see that it doesn't make much sense this way. and looks a bit confusing.

**Pull request merged**